astrapy/db.py

```diff
@@ -44,7 +46,7 @@ def __init__(
 
```

Comment:
we are ok that the "direct" instantiation of AstraDBCollection (= without passing a pre-created AstraDB) does not offer the advanced params (api_path and api_version) ?
I am ok with that - it's a shortcut after all.

Reply:
That was kind of my feeling too!